backend/src/schema/resolvers/helpers/existingEmailAddress.js

```diff
@@ -0,0 +1,26 @@
+import { UserInputError } from 'apollo-server'
+export default async function alreadyExistingMail(_parent, args, context) {
```

Comment:
Not on purpose. Next time I touch this file I will go with `existingEmailAddress`. This makes sense especially because we have multiple nodes for similar things now.